### PR TITLE
Course victory modal

### PR DESCRIPTION
**Pull request by sderickson**
Creates a new victory modal, which is only set up for when you’re playing a level within a course. TODOs stemming from this first step:

* Finish rest of migration from VictoryModal and HeroVictoryModal to CourseVictoryModal (and rename back to VictoryModal).
* Animate the new item icon (@nwinter need to discuss animations with you).
* Create static or dynamic graphics for well-parchment, img-prize and label-banner.

### CODE REVIEW

File: app/templates/play/level/modal/progress-view.jade [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        .col-sm-5
+          .well.well-sm.well-parchment
+            h3.text-uppercase Next Level:
+            h2.text-uppercase= i18n(view.nextLevel.attributes, 'name')
+            
+            p= i18n(view.nextLevel.attributes, 'description')
+        
+    .row
+      .col-sm-5.col-sm-offset-2
+      //  button#continue-btn.btn.btn-illustrated.btn-default.btn-block.btn-lg.text-uppercase View Leaderboards
```
Comment (differentmatt) [inline app/templates/play/level/modal/progress-view.jade]: Is this here for future work?  If so, probably should comment this sort of thing.

**Comment by differentmatt** [comment]
I get 0/0 progress for many levels.  Any idea what's going on here?

Production example: http://localhost:3000/play/level/oasis?course=56462f935afde0c6fd30fc8c&course-instance=56462fac4e6f678605e8b197

**Comment by differentmatt** [comment]
Code looks good.  Only open issue for me is the 0/0 progress.

**Comment by nwinter** [comment]
I saw progress working sometimes, but not if I open the modal again after escaping out of it on a particular level.

Another thing to note is that we will have to figure out how to change the way we "award" items for courses, since the levels in which it would make sense to show certain new loot are different than in the campaign, and we want to award more things (that players would normally buy, like armor) that we won't award in campaigns. Maybe instead of reading the new items from the achievements when it's a course level, we read them directly from a spot in the level config instead that only applies to course levels?

**Comment by sderickson** [comment]
@nwinter No problem, merged in, fixed the conflict, and will squash out the commit on a rebase. I suppose that's a benefit of making a pull request from a fork, only I can edit it.

How do course levels currently indicate what items are used? Would you want this level config to be added as part of this branch or put on a todo list?

**Comment by nwinter** [comment]
We can fix it separately; the way that it loads items from the achievements that actually grant them from the campaign version is okay for now. I'll add a TODO to change the way we are storing those for Courses.

**Comment by sderickson** [comment]
@differentmatt Can you check to see if the 0/0 issue is still happening? I was able to reproduce the issue how Nick described, closing and re-opening the modal, and fixed that, but it sounded like it have been more persistent for you.

**Comment by differentmatt** [comment]
I don't see 0/0 issues with the latest changes.

**Pull request merged**